packages/base-driver/test/basedriver/driver-e2e-tests.js

```diff
@@ -1,8 +1,5 @@
 import _ from 'lodash';
-import { BaseDriver, server, routeConfiguringFunction, DeviceSettings, errors } from '../../lib';
-import {
-  MJSONWP_ELEMENT_KEY, W3C_ELEMENT_KEY
-} from '../../lib/constants';
+import { BaseDriver, server, routeConfiguringFunction, DeviceSettings } from '../../lib';
```

Comment:
can we remove stuff from `lib/constants` too?

Reply:
they are used elsewhere. but it's ok, they just import the value from appium support in any case.